chapters/en/Unit 2 - Convolutional Neural Networks/yolo.mdx

```diff
@@ -0,0 +1,58 @@
+# YOLO 
+
+## A short introduction to Object Detection
+
+Convolutional Neural Networks has taken a big step towards the image classifcation problem. But there remains
+another big task to solve. Object detection. Object detection not only requires to categorize the object from the image but also to accurately predict the location (in this case the co-ordinates of the bounding boxes of the object) from the image. Here came the big breakthrough of YOLO. Before delving into the depth of YOLO, lets get through the history of object detection algorithms using CNN.
+
+
+## RCNN , Fast RCNN , Faster RCNN
+
+### RCNN
+RCNN is the simplest way possible to use Convolutional neural network for Object detection. to be as simple as possible , the basic idea is to 
+detect a "region" and then use CNN to classify the region . So this is a multi-step process . Based on this idea the RCNN paper was shared in 2012[1]
+
+The RCNN uses following steps , 
+
+- Use Selective Search algorithm to select a region. 
```

Comment:
Because they are steps I would propose to put them as 1) ,2) instead of Bullet points.